src/main/java/me/lokka30/microlib/wholes/WholeTitle.java

```diff
@@ -23,7 +23,7 @@
  * @version 3.2.0
  * @since 1.0.3 -ALPHA
  */
-@SuppressWarnings("unused")
+@SuppressWarnings({"unused", "deprecation"})
 public class WholeTitle {
```

Comment:
I don't like the use of NMS in this class when we can access the deprecated sendTitle method for older servers. I'll work on this unless you insist